src/YawPitchControl/input/FusionPoseSensor.js

```diff
@@ -97,28 +104,72 @@ export default class FusionPoseSensor extends Component {
 		}
 
 		this.trigger("change", {quaternion: orientation});
+		// console.log("orientation", orientation);
 	}
 	getOrientation() {
-		// Convert from filter space to the the same system used by the
-		// deviceorientation event.
-		const orientation = this.filter.getOrientation();
-
-		if (!orientation) {
-			return null;
+		let orientation;
+
+		// Hack around using deviceorientation instead of devicemotion
+		if (this.deviceMotion.isWithoutDeviceMotion && this._deviceOrientationQ) {
+			// We must rotate 90 degrees on the Y axis to get the correct
+			// orientation of looking down the -Z axis.
+			this.deviceOrientationFixQ = this.deviceOrientationFixQ || (function() {
+				const z =
+					new MathUtil.Quaternion().setFromAxisAngle(new MathUtil.Vector3(0, 0, -1), 0);
+				const y =
+					new MathUtil.Quaternion().setFromAxisAngle(new MathUtil.Vector3(0, 1, 0), Math.PI / 2);
+
+				return z.multiply(y);
+			})();
+			orientation = this._deviceOrientationQ;
+			const out = new MathUtil.Quaternion();
+
+			out.copy(orientation);
+			out.multiply(this.filterToWorldQ);
+			out.multiply(this.resetQ);
+			out.multiply(this.worldToScreenQ);
+			out.multiplyQuaternions(this.deviceOrientationFixQ, out);
+
+			// return quaternion as glmatrix quaternion object
+			const out_ = quat.fromValues(
+				out.x,
+				out.y,
+				out.z,
+				out.w
+			);
+
+			return quat.normalize(out_, out_);
+		} else {
+			// Convert from filter space to the the same system used by the
+			// deviceorientation event.
+			orientation = this.filter.getOrientation();
+
+			if (!orientation) {
+				return null;
+			}
+
+			// Predict orientation.
+			this.predictedQ =
+				this.posePredictor.getPrediction(orientation, this.gyroscope, this.previousTimestampS);
```

Comment:
@happyhj How about use `_convertFusionToPredicted` as before?

Reply:
OK, I'll take that.